LibuvSharp/UVStream.cs

```diff
@@ -211,7 +212,58 @@ public void Write(ArraySegment<byte> data, Action<Exception> callback)
 			Ensure.Success(r);
 		}
 
-		public void Shutdown(Action<Exception> callback)
+        public void Write(IList<ArraySegment<byte>> buffers, Action<Exception> callback)
```

Comment:
Oh, I now found a reason to use `IList`: There is no `Count` property on `IEnumerable`, hence `IList` saves us one iteration for counting ourselves.